demo/cluster/verify-kyverno-policy.sh

```
@@ -170,6 +168,53 @@ else
+        pass "CLI SA: ConfigMap blocked by Kyverno (not RBAC)"
+        if echo "${CLI_REJECT_OUTPUT}" | grep -q "cluster-whisperer-cli-resource-allowlist"; then
+            pass "CLI SA: Rejection references the CLI policy"
+        else
+            fail "CLI SA: Rejection did not reference CLI policy. Output: ${CLI_REJECT_OUTPUT}"
+        fi
+    else
+        fail "CLI SA: ConfigMap was NOT blocked. Output: ${CLI_REJECT_OUTPUT}"
+    fi
+
+    info "Test 6: ManagedService creation as cluster-whisperer-cli SA should be allowed by Kyverno..."
+    local cli_allow_exit=0
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)